sqlmesh/core/config/loader.py

```diff
@@ -27,7 +28,9 @@ def load_configs(
     config = config or "config"
 
     absolute_paths = [
-        Path(t.cast(t.Union[str, Path], path)).absolute() for path in ensure_list(paths)
+        Path(t.cast(t.Union[str, Path], p)).absolute()
+        for path in ensure_list(paths)
+        for p in (glob.glob(str(path)) or [str(path)])
```

Comment:
`glob.glob()` returns `None` for absolute paths

Reply:
Did some digging and it was not the absolute paths.

The issue is that glob.glob returns an empty list if the pattern doesn't match anything (This was the case for a few pytest cases). The `load_configs` function would return an unexpected empty dictionary and causing downstream issues. 

Adding `or [str(path)]` will the correct downstream invalidation of the path/config.

Reply:
I see, seems reasonable.